### PR TITLE
updates to the metrics collection for quorum

### PR DESCRIPTION
**Pull request by joshuafernandes**
Signed-off-by: Joshua Fernandes <joshua.fernandes@consensys.net>

## Pull Request Description
updates to the metrics collection for quorum and updating quorum dashboard images

### CODE REVIEW

**Review by bgravenorst — APPROVED** [review]
@joshuafernandes, let me know if you agree with my changes, then we can merge.

File: docs/HowTo/Monitor/monitoring.md [hunk 1]
```diff
@@ -1,55 +1,71 @@
...
-  --metrics.influxdb.username value   Username to authorize access to the database (default: "test")
-  --metrics.influxdb.password value   Password to authorize access to the database (default: "test")
-  --metrics.influxdb.tags value       Comma-separated InfluxDB tags (key/values) attached to all measurements (default: "host=localhost")
-```
+Use `--metric.influxdb` and associated [command line options] to push metrics data to [InfluxDB].
 
-!!!warning
-    Do not expose `pprof` HTTP end point to public Internet!
-    This end point can be used to trigger resource intensive operations.
+!!! note "Exerpt of geth command line options"
```
Comment (alexandratran) [inline docs/HowTo/Monitor/monitoring.md:32]:
```suggestion
!!! note "Excerpt of geth command line options"
```

File: docs/HowTo/Monitor/monitoring.md [hunk 2]
```diff
@@ -1,55 +1,71 @@
...
+      --metrics.influxdb.endpoint value   InfluxDB API endpoint to report metrics to (default: "http://localhost:8086")
+      --metrics.influxdb.database value   InfluxDB database name to push reported metrics to (default: "geth")
+      --metrics.influxdb.username value   Username to authorize access to the database (default: "test")
+      --metrics.influxdb.password value   Password to authorize access to the database (default: "test")
+      --metrics.influxdb.tags value       Comma-separated InfluxDB tags (key/values) attached to all measurements (default: "host=localhost")
+    ```
+
+## Visualize collected data
+
+GoQuorum metrics data can be visualised with many dashboard tools. We provide a
```
Comment (alexandratran) [inline docs/HowTo/Monitor/monitoring.md:53]:
```suggestion
GoQuorum metrics data can be visualized with many dashboard tools. We provide a
```

**Comment by joshuafernandes** [comment]
> @joshuafernandes, let me know if you agree with my changes, then we can merge.

Good to go @bgravenorst :)

**Pull request merged**